gemfiles/rails4_2.gemfile

```diff
@@ -3,6 +3,11 @@ source 'https://rubygems.org'
 gem 'arel', '~> 6.0'
 
 group :development, :test do
+  rb_version = Gem::Version.new(RUBY_VERSION)
+  if rb_version > Gem::Version.new('2.6') && rb_version < Gem::Version.new('2.8')
```

Comment:
Yes `Gem::Version` takes care of semver:

```
irb(main):001:0> a = Gem::Version.new('2.6.1')
=> Gem::Version.new("2.6.1")
irb(main):002:0> b = Gem::Version.new('2.6')
=> Gem::Version.new("2.6")
irb(main):003:0> a < b
=> false
irb(main):004:0> b < a
=> true
```

And yeah I want to restrict it to `2.7` because this dependency issue pops up just in that case (yay bundler).

Reply:
I believe you exactly proved my point, not yours.  You don't mean `2.6 <`, you mean `2.7 <=`.